src/uct/ib/base/ib_iface.c

```diff
@@ -81,6 +81,16 @@ ucs_config_field_t uct_ib_iface_config_table[] = {
    "Number of send WQEs for which completion is requested.",
    ucs_offsetof(uct_ib_iface_config_t, tx.cq_moderation), UCS_CONFIG_TYPE_UINT},
 
+#ifdef HAVE_DECL_IBV_EXP_CQ_MODERATION
```

Comment:
not sure if we should hide config parameters on non-support EXP_MODERATION

Reply:
I'd hide, like it is done now